### ISSUE TITLE
Set up E2E Llama LoRA training

### ISSUE THREAD

**Issue by vladimirjovanovicTT** [tenstorrent/tt-blacksmith#73]
Set up end-to-end LoRA training through tt-forge-fe (using the dataset and hyperparams selected previously).

We want to be able to compare training loss/artifacts to the ones we got from GPU training.
It is ok to have this done in a hacky way; we will add infra for experiment comparison as the stack matures.
